scripts/tests/test_safe_template_linter.py

```diff
@@ -2,45 +2,14 @@
 Tests for safe_template_linter.py
 """
 from ddt import ddt, data
-import mock
-import re
-from StringIO import StringIO
 import textwrap
 from unittest import TestCase
 
 from ..safe_template_linter import (
-    _process_os_walk, FileResults, MakoTemplateLinter, UnderscoreTemplateLinter, Rules
+    FileResults, MakoTemplateLinter, UnderscoreTemplateLinter, Rules
 )
 
 
-class TestSafeTemplateLinter(TestCase):
-    """
-    Test some top-level linter functions
-    """
-
-    def test_process_os_walk_with_includes(self):
```

Comment:
Is the functionality in `_process_os_walk` tested elsewhere, without handling includes? Or is that unneeded?